sunnypilot/selfdrive/controls/lib/nnlc/nnlc.py

```diff
@@ -126,7 +127,7 @@ def update_neural_network_feedforward(self, CS, params, calibrated_pose) -> None
     self._ff = self.model.evaluate(nn_input)
 
     # apply friction override for cars with low NN friction response
-    # TODO-SP: verify with twilsonco if this change is appropriate
     if self.model.friction_override:
       self._pid_log.error += self.torque_from_lateral_accel(LatControlInputs(0.0, 0.0, CS.vEgo, CS.aEgo), self.torque_params,
```

Comment:
This line should be removed. (It does nothing anyhow since the input lat accel is zero and there's no roll compensation or lateral jerk input either.)